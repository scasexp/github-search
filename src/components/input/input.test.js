import React from 'react';
import renderer from 'react-test-renderer';
import InputApp from './index';

test('MatchSnap', () => {
    const component = renderer.create(
      <InputApp />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

