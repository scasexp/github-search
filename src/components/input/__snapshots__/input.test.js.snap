// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MatchSnap 1`] = `
<input
  onChange={[Function]}
/>
`;
